src/clojure/nrepl/threads.clj

```diff
@@ -0,0 +1,4 @@
+(ns nrepl.threads)
+
+(def ^{:dynamic true
```

Comment:
I don't think that a separate namespace is warranted for just one configuration value, so I'd probably move this to the session namespace.